src/main/archetype/ui.content/src/main/content/jcr_root/content/experience-fragments/__contentFolderName__/language-masters/en/.content.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    cq:adobeTargetExportFormat="html"
+    jcr:primaryType="sling:OrderedFolder"
+    jcr:title="${language_country.split('_').get(0)}">
```

Comment:
Isn't the title = "en" instead as the path is ".../language-masters/en"?

Reply:
The path is moved according to the input of the users input. This is done in the post-generation-groovy script in https://github.com/adobe/aem-project-archetype/pull/252/files#diff-346cccad6cc3468e44003d42921faec1R50